<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MountCS</class>
 <widget class="QMainWindow" name="MountCS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>700</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Coldshield Mount Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/sbf40010/SHARED/DEWAR/PUBLIC/DF/177/calcSupport/LM.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelHeader">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>12</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>MS-177 Coldshield Mount Calculator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelImg1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>190</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/sbf40010/SHARED/DEWAR/PUBLIC/DF/177/calcSupport/cs1.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelImg2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/sbf40010/SHARED/DEWAR/PUBLIC/DF/177/calcSupport/cs2.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelImg3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>190</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/sbf40010/SHARED/DEWAR/PUBLIC/DF/177/calcSupport/cs3.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelImg4">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>475</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/sbf40010/SHARED/DEWAR/PUBLIC/DF/177/calcSupport/cs4.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Control Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter -A- to (4) Coldshield plateau heights</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>315</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>+ Nominal Coldfilter thickness</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>- Datum -A- to FPA (optical) centerline</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>475</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+ Expected Coldfilter Bondline</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>s/b 5.5973 Â± .0040</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputHeight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditControl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>XXXXXXXXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCS">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCF">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>315</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFPA">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoad">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>585</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>615</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>s/b â¤ 0.0020</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>625</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputParallel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>615</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Expected Optical Center to Exit Aperture Height</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>615</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Datum -A- to CS Parallelism</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlateau1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlateau2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlateau3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlateau4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Datum -A- to Coldshield height</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dewar Serial Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSerial">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>XXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBL">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>475</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.0015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.0025</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.003</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionScreenShot"/>
    <addaction name="actionShowNotepad"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowCalc"/>
    <addaction name="actionShowBuild"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear Data</string>
   </property>
  </action>
  <action name="actionScreenShot">
   <property name="text">
    <string>Get Screenshot</string>
   </property>
  </action>
  <action name="actionShowNotepad">
   <property name="text">
    <string>Show Notepad</string>
   </property>
  </action>
  <action name="actionShowCalc">
   <property name="text">
    <string>Show Calculations</string>
   </property>
  </action>
  <action name="actionShowBuild">
   <property name="text">
    <string>Show Build History</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditControl</tabstop>
  <tabstop>pushButtonLoad</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>lineEditPlateau2</tabstop>
  <tabstop>lineEditPlateau1</tabstop>
  <tabstop>lineEditPlateau4</tabstop>
  <tabstop>lineEditPlateau3</tabstop>
  <tabstop>pushButtonCalc</tabstop>
  <tabstop>lineEditCS</tabstop>
  <tabstop>lineEditCF</tabstop>
  <tabstop>lineEditFPA</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>MountCS</receiver>
   <slot>loadData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>MountCS</receiver>
   <slot>clearData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCalc</sender>
   <signal>clicked()</signal>
   <receiver>MountCS</receiver>
   <slot>calculateData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>MountCS</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowNotepad</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>showNotepad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowCalc</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>showCalculations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowBuild</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>showBuildData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenShot</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>getScreenShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>loadData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>clearData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MountCS</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadData()</slot>
  <slot>saveData()</slot>
  <slot>clearData()</slot>
  <slot>calculateData()</slot>
  <slot>getScreenShot()</slot>
  <slot>showNotepad()</slot>
  <slot>showCalculations()</slot>
  <slot>showBuildData()</slot>
  <slot>showAbout()</slot>
 </slots>
</ui>
